gallery/how_to/deploy_models/deploy_model_on_adreno.py

```diff
@@ -0,0 +1,281 @@
+"""
+Deploy the Pretrained Model on Adreno
+=======================================
+**Author**: Daniil Barinov
+
+This article is a step-by-step tutorial to deploy pretrained MXNet VGG16 model on Adreno (on different precisions).
+
+For us to begin with, MXNet package must be installed.
+
+A quick solution is to install it via pip:
+
+.. code-block:: bash
+
+  pip install mxnet
```

Comment:
this is quite bad choice of the model. It has benefits as ease-of-use, but tvm has issue with latest python and latest mxnet: https://github.com/apache/tvm/issues/11452